lambda_functions/source/cw-helper/organizations.py

```diff
@@ -26,8 +27,11 @@ def get_org_id():
     management_account_session = boto3.Session()
     org_client = management_account_session.client("organizations")
     response = org_client.describe_organization()["Organization"]
+    organization_id = response["Id"]
+    parents = org_client.list_parents(ChildId=EVENTBUS_ACCOUNT).get('Parents')
```

Comment:
The Execution role should be allowed to Access `organizations:ListParents`  at https://github.com/aws-ia/cfn-abi-crowdstrike-fcs/blob/78ad994a15d565da93c73d86e018ec759a3e7072/templates/crowdstrike_init_stack.yaml#L1315